.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 	},
 	extends: [
 		'eslint:recommended',
-		'plugin:react/recommended',
+		'plugin:react/jsx-runtime',
```

Comment:
Can I ask about he rationale behind this?

Reply:
The import React from 'react'; is removed, therefor we have to tell the linter about this (we are using the new jsx transform) See comment : 

```If you are using the [new JSX transform from React 17](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports), extend [react/jsx-runtime](https://github.com/jsx-eslint/eslint-plugin-react/blob/c8917b0885094b5e4cc2a6f613f7fb6f16fe932e/index.js#L163-L176) in your eslint config (add "plugin:react/jsx-runtime" to "extends") to disable the relevant rules.```

source: https://github.com/jsx-eslint/eslint-plugin-react

Reply:
Cool, thanks for clarifying :pray: